docs/src/dev_guide/inputs.md

```diff
@@ -0,0 +1,337 @@
+
+ - **index_z.csv**: Balancing Area zones as modeled in ReEDS. Refer to the [model regions map](https://www.nrel.gov/analysis/assets/pdfs/reeds-model-regions-map.pdf).
+
+ - **index_sector.csv**: Customer high level groups. Example values are `Residential, Commercial, Industrial`.
+
+ - **index_h_sector_mapping.csv**: File that maps each zone in index\_z to a group in index\_sector. It conisits of two columns, `index_z` and `index_h`.
+
+### Agent-Specfic Indices
+
+#### Regulator
+
+ - **index_rate_tou.csv**: Index for supported time-of-use rates. Example values are `Off-Peak, On-Peak, Mid-Peak`.
+
+ - **tou_rate_structure_\$(tou_suffix).csv**: Time-of-use rate for each representative hour and representative day combination. The column names are `index_d`, `index_t` and `index_rate_tou`. Here, the input file selected depends on the `tou_suffix` specified in the config.yaml file under `regulator_options`.
+
+#### VIU/IPP
+
+ - **index_k_existing.csv**: Existing bulk generation technologies. Consists of a single row containing the existing technolgies, for e.g., `coaloldscr`, `gas-ct`, `nuclear`, `wind-ofs` etc.
+
+ - **index_k_new.csv**: Potential bulk generation technologies. Consists of a single row containing the new technolgies, for e.g., `coaloldscr`, `gas-ct`, `nuclear`, `wind-ofs` etc.
+
+ - **index_rps.csv**: [Renewable Potfolio Standard (RPS)](https://www.nrel.gov/state-local-tribal/basics-portfolio-standards.html) qualified technologies. Consists of a single row containing qualified technolgies, for e.g., `biopower`, `wind-ofs` etc.
+
+ - **index_stor_existing.csv**: Existing bulk storage technologies. Consists of a single row containing the existing technolgies, for e.g., `battery_1`, `battery_2` etc.
+
+ - **index_stor_new.csv**: Potential bulk storage technologies. Consists of a single row containing the existing technolgies, for e.g., `battery_1`, `battery_2` etc.
+
+ - **index_l.csv**: Transmission lines. Each line is represented as `\$(origin_zone)||\$(destination_zone)`, for e.g., `p100||p101`.
+
+ - **index_p.csv**: Individual independent power producers (IPP). Cosnsists of a single row of IPP names.
+
+#### Customer
+
+ - **index_m.csv**: Behind-The-Meter (BTM) technologies. Eaxample values are `BTMPV, BTMStorage`.
+
+## Parameters
+
+### Model Parameters
+
+ - **Omega.csv**: Number of days per representative day. Column names are the representative days in index_d and the values are the number of days for that representative day.
+
+ - **Gamma.csv**: Number of customers of type in index\_h at zone in index\_z. Column names are index\_z, followed by the customer groups (for e.g., `index_z, Commercial, Residential, Industrial`).
+
+ - **Year.csv**: Column names and values are the representative years in index_y.
+
+ - **Time.csv**: Number of days per representative hour. Column names are the representative hours in index_t and the values are the number of hours for that representative hour.
+
+ ### Hourly Resource Data
```

Comment:
AvailabilityOld.csv and AvailabilityNew.csv should also be listed here. They are essentially duplicates of AvailabilityNewIPP.csv and AvailabilityOldIPP.csv except without index\_p.